src/tests/test_files/dynamo0p3/enforce_operator_bc_kernel_mod.F90

```diff
@@ -1,3 +1,37 @@
+! -----------------------------------------------------------------------------
+! BSD 3-Clause License
+!
+! Copyright (c) 2017, Science and Technology Facilities Council
+! However, it has been created with the help of the GungHo Consortium,
```

Comment:
I think the original license should be at the top. If you have modified the code then adding BSD is good but could you please remove lines 5,6,7 and also add modified by A. Porter STFC.